app_user/views.py

```diff
@@ -87,4 +100,96 @@ def changePassword(request):
             return HttpResponseRedirect(reverse('app:startPage'))
     else:
         form = PasswordChangeForm(user=request.user)
-    return render_to_response('app/change_password.html', {'form': form}, RequestContext(request))
\ No newline at end of file
+        action = 'http://' + HttpRequest.get_host(request) + reverse('app_user:change_password')
+    return render_to_response('app_user/password.html', {'form': form, 'action': action}, RequestContext(request))
+
+def sendPasswordResetEmail(request):
+	email = request.POST.get('email', '')
+	user=User.objects.filter(email=email).first()
+
+	result = None
+
+	if user is not None:
+		subject = "Password Zurücksetzen book²"
+
+		confirmId = get_random_string(length=32)
+		result = PasswordReset.objects.filter(user=user).first()
+
+		# create entry if not exists
+		if result is None:
+ 			result = PasswordReset()
+
+		# add/edit entry
+		result.uuid = confirmId
+		result.user = user
+		result.save()
+
+		link = 'http://' + HttpRequest.get_host(request) + reverse('app_user:new_password', kwargs={'uuid': confirmId})
+		message = 'Password Zurücksetzen in book².<br> Zur Bestätigung des Password Zurücksetzens betätigen Sie bitte folgenden Link: <a href="' + link + '">Bestätigen</a><br>Sollten Sie den Link nicht nutzen könnten dann kopieren Sie bitte folgende URL in Ihren Browser:<br> ' + link + '<br><br>Wir wünschen Ihnen viel Spaß beim Shoppen<br>Ihr book² team'
```

Comment:
Ich habe vor kurzem in der Aktivierungsmail die Anrede auf "Hallo $vorname $nachname" geändert, da ich das eleganter fand als die vielen Schrägstriche - zwecks Einheitlichkeit evtl. auch hier diese Anrede nehmen. :3